### ISSUE TITLE
Substring matches

### ISSUE THREAD

**Issue by jasonvarga** [mmanos/laravel-search#31]
Is it possible to get a hit for a partial word?

For example, if I search for `wild`, I'd like to see results containing `wilderness`.

**Comment by dmyers** [comment]
Yeah we need to find a way to implement fuzzy matching somehow across all the drivers. Zend and ElasticSearch support it and I imagine Algolia does too.
